cypress/e2e/patient_spec/patient_prescription.cy.ts

```diff
@@ -0,0 +1,49 @@
+import { PatientEncounter } from "@/pageObject/Patients/PatientEncounter";
+import { FacilityCreation } from "@/pageObject/facility/FacilityCreation";
+
+const facilityCreation = new FacilityCreation();
+const patientEncounter = new PatientEncounter();
+
+describe("Patient Prescription Management", () => {
+  beforeEach(() => {
+    cy.loginByApi("devnurse");
```

Comment:
```suggestion
    cy.loginByApi("devnurse");
```

- create a new user and update it in fixtures then use that id here